README.md

```diff
@@ -159,6 +159,7 @@ Enable the rules that you would like to use.
 * [react/jsx-one-expression-per-line](docs/rules/jsx-one-expression-per-line.md): Limit to one expression per line in JSX
 * [react/jsx-curly-brace-presence](docs/rules/jsx-curly-brace-presence.md): Enforce curly braces or disallow unnecessary curly braces in JSX
 * [react/jsx-pascal-case](docs/rules/jsx-pascal-case.md): Enforce PascalCase for user-defined JSX components
+* [react/jsx-prefer-fragment-shorthand](docs/rules/jsx-prefer-fragment-shorthand.md): Enforce standard or shorthand syntax for React fragments
```

Comment:
maybe “prefer” shouldn’t be in the rule name, since that’s only one of the settings

Reply:
`jsx-fragment-syntax` of either `'shorthand'` or `'standard'`? (Also not sure what’s the better default)

Reply:
Technically React.Fragment isn’t syntax, it’s api - maybe `jsx-fragments` with either “syntax” or “element” or something.

I’d say the default should probably be “syntax” even though that requires babel 7.

Reply:
Ah, I see what you mean 👍